src/actions/ticket-actions.js

```diff
@@ -183,6 +183,10 @@ const parseFilters = (filters, term = null) => {
     filter.push("exclude_is_printable_free_unassigned==1");
   }
 
+  if (filters?.noPromocodeFilter) {
+    filter.push("not_promo_code_id==1");
```

Comment:
@tomrndom filter is called has_promo_code
change at summit api is deployed at dev